### ISSUE TITLE
Tipping button in pages.

### ISSUE THREAD

**Issue by hackrush01** [lbryio/lbry-app#355]
Story: As a publisher I would like if the content page has a tipping button, so that people could send me some LBCs without first resolving my content uri, then searching for my address, and then tipping me either via cli or UI.

**Comment by kauffj** [comment]
The tip can be sent to the same address of the payment, or the address of the claim if no payment details are specified.

**Comment by tzarebczan** [comment]
Possibly related to https://github.com/lbryio/lbry-app/issues/247 based on team discussions.

**Comment by tzarebczan** [comment]
I forget where the conversation took place, but we mentioned using claim supports for tipping/karma that then could be redeemable (maybe a timelock feature?). 

Just wanted to mention it here so the idea doesn't get lost. The duplicate claim_id support issue would need to be fixed first (https://github.com/lbryio/lbry/issues/795)

**Comment by kauffj** [comment]
I'm guessing the first iteration of tips will just be sends to the address since I don't think issuing tips as supports is possible via the existing LBRY API. (Please correct this @jackrobison)

**Comment by tzarebczan** [comment]
When we tested tipping, the tipped amount shows up in wallet but gets inserted as a support, which is not spendable.  This issue may need to be expanded so that it's clearer to the users what is happening (WHERE IS MY LBC??!?, WHY IS MY BALANCE INCORREC?T?!?): https://github.com/lbryio/lbry-app/issues/379

Also, claim_abandon support: https://github.com/lbryio/lbry/issues/795 will be needed to fully take advantage of the tips.

**Comment by hackrush01** [comment]
After having a discussion on hangouts regarding the working of the `send_claim_tip` command, there were several other issues that prevented this from working properly. One of which is pointed in the last comment. As for the solution part, it would need a new kind of wallet and better claim management system to get it working properly. Right now the feature is not working in the way it was intended in the daemon itself. So this is more of the daemon issue but that will be blocking he PR here.

If we want this feature to be out there now, then it has to be done in the more straight-forward way, like a direct wallet transaction.

> tl;dr `send_claim_tip` requires deeper daemon changes to work properly. A basic tip/support implementation is possible.

**Comment by tzarebczan** [comment]
@jackrobison may want to chime in here, but this was what we discussed on the call this morning.  If we want tipping ASAP, it just needs to be a simple LBC send tx as opposed to the new supports mechanism.

**Comment by kauffj** [comment]
I caught up on this and discussed with Jack. I think we can still move forward with this. @hackrush01 let's sync on Slack.
